src/lib.rs

```diff
@@ -79,9 +83,11 @@ pub fn create_ecx<'mir, 'tcx: 'mir>(
     let mut ecx = InterpretCx::new(
         tcx.at(syntax::source_map::DUMMY_SP),
         ty::ParamEnv::reveal_all(),
-        Evaluator::new(config.validate, config.seed),
+        Evaluator::new(config.validate),
     );
 
+    ecx.memory_mut().extra.rng = config.seed.map(StdRng::seed_from_u64);
```

Comment:
Can't we set things up such that `MemoryExtra` gets created with the right RNG?

Reply:
I didn't look it up. Let me see

Reply:
Could you add a "FIXME: InterpretCx::new should take an initial MemoryExtra" comment?